ibc-clients/ics07-tendermint/types/src/client_state.rs

```diff
@@ -283,9 +284,6 @@ impl TryFrom<RawTmClientState> for ClientState {
             .try_into()
             .map_err(|_| Error::MissingLatestHeight)?;
 
-        // In `RawClientState`, a `frozen_height` of `0` means "not frozen".
-        // See:
-        // https://github.com/cosmos/ibc-go/blob/8422d0c4c35ef970539466c5bdec1cd27369bab3/modules/light-clients/07-tendermint/types/client_state.go#L74
         let frozen_height = raw.frozen_height.and_then(|h| Height::try_from(h).ok());
```

Comment:
`None` and `RawHeight { number: 0, height: 0 }` - these two cases are treated similarly here. Should we allow these?

Reply:
Per our discussion: [6d9416d](https://github.com/cosmos/ibc-rs/pull/1064/commits/6d9416dc2f4b13e40eaceae25dd032ab92a7d2bc)